client/nuxt.config.js

```diff
@@ -1,6 +1,6 @@
 const pkg = require('./package.json')
 
-const routerBasePath = process.env.ROUTER_BASE_PATH || ''
+const routerBasePath = process.env.ROUTER_BASE_PATH || '/audiobookshelf'
```

Comment:
When the `ROUTER_BASE_PATH` env variable is set to an empty string in order to restore the old behavior then it overrides it to `/audiobookshelf`.

Is that expected? I would have thought that the `/audiobookshelf` fallback should only have been only used in the case where the `ROUTER_BASE_PATH` env variable is undefined.

Reply:
https://github.com/advplyr/audiobookshelf/pull/3958